dist/pythonlibs/riotctrl_shell/ieee802154.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2019-20 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import re
+
+from riotctrl.shell import ShellInteraction
+
+# ==== ShellInteractions ====
+
+class Config_phy(ShellInteraction):
+    def config_phy(self, phy_mode, channel, tx_pow, timeout=None, async_=False):
+        cmd = "config_phy {phy_mode} {channel} {tx_pow}" \
+              .format(phy_mode=phy_mode, channel=channel, tx_pow=tx_pow)
+        
+        try:
+            res = self.cmd(cmd, timeout=timeout, async_=async_)
+        except Exception as e:
+            print(str(e))
+            print("Exception")
+        
+        if "Success" not in res:
+            raise RuntimeError(res)
+
+        if str(channel) not in res:
+            raise RuntimeError(res)
+
+class Print_addr(ShellInteraction):
+    def print_addr(self, timeout=None, async_=False):
+        cmd = "print_addr" \
+              .format()
+
+        try:
+            res = self.cmd(cmd, timeout=timeout, async_=async_)
+        except Exception as e:
+            print(str(e))
+            print("Exception")
```

Comment:
Redundant output and cast usage. How about just:

```suggestion
            print("Error:", e)
```

Also, never just catch `Exception` but specific ones. [`pylint`](https://pylint.pycqa.org/en/latest/) or [`flake8`](https://flake8.pycqa.org/en/latest/) will tell you as much.